net/tlsdial/blockblame/blockblame.go

```diff
@@ -0,0 +1,89 @@
+// Package blockblame blames specific firewall manufacturers for blocking Tailscale,
+// by analyzing the SSL certificate presented when attempting to connect to a remote
+// server.
+package blockblame
+
+import (
+	"crypto/x509"
+	"strings"
+)
+
+// Manufacturer represents a firewall manufacturer that may be blocking Tailscale.
+type Manufacturer struct {
+	// Name of the firewall manufacturer, e.g. "Fortinet"
+	CompanyName string
+	// List of indicators that can be used to identify the manufacturer
```

Comment:
// Indicators is ...

But how about instead:

    // Match reports whether a cert matches this Manufacturer.
   Match func(*x509.Certificate) bool

Then you can have helper functions `func any` and `hasOUSubstr` etc.

Reply:
done